Modules/Libs/QuestieLib.lua

```diff
@@ -535,3 +535,114 @@ function QuestieLib.tunpack(tbl)
 
     return recursion(1)
 end
+
+
+local frameObject = nil
+if _G["QuestLogObjectivesText"] then -- classic
+    frameObject = _G["QuestLogObjectivesText"] 
+elseif _G["QuestInfoObjectivesText"] then -- Wotlk Classic
+    frameObject = _G["QuestInfoObjectivesText"]
+end
+--Part of the GameTooltipWrapDescription function
+local objectiveFontString = UIParent:CreateFontString("questieObjectiveTextString", "ARTWORK", "QuestFont")
+objectiveFontString:SetWidth(frameObject:GetWidth() or 275) --QuestLogObjectivesText default width = 275
+objectiveFontString:SetHeight(0);
+objectiveFontString:SetPoint("LEFT");
+objectiveFontString:SetJustifyH("LEFT");
+---@diagnostic disable-next-line: redundant-parameter
+objectiveFontString:SetWordWrap(true)
+objectiveFontString:SetVertexColor(1,1,1, 1)--Set opacity to 0, even if it is shown it should be invisible
+local font, size = frameObject:GetFont()
+--Chinese? "Fonts\\ARKai_T.ttf"
+objectiveFontString:SetFont(font, size);
+objectiveFontString:Hide()
+
+---Emulates the wrapping of a quest description
+---@param line string @The line to wrap
+---@param prefix string @The prefix to add to the line
+---@param combineTrailing boolean @If the last line is only one word, combine it with previous? TRUE=COMBINE, FALSE=NOT COMBINE, default: true
+---@param splitOnDot boolean @Should we add a linebreak if a dot appears thats not at the end of a line TRUE=NEW ROW, FALSE=NO NEW ROW, default: true
+---@param desiredWidth number @Set the desired width to wrap, default: 275
+---@return table[] @A table of wrapped lines
+function QuestieLib:TextWrap(line, prefix, combineTrailing, splitOnDot, desiredWidth, questid)
+    if(objectiveFontString:IsVisible()) then print("TextWrap already running... strange, look into pooling the FontString ugh") end
```

Comment:
Shouldn't this be a real debug print or something?

Reply:
There are two ways to think here, this should NEVER happen. But if it does it is actually really bad. So if it happens I kind of want to know about it.
But as long as no one changes the function to add a coroutine yield in the middle of it the code is basically "dead".
If we do a regular debug, we might as well remove the entire line for performance :)

Reply:
Why would `Questie:Error("TextWrap already running... Please report this on GitHub or Discord.")` be worse?

Reply:
Ehh didn't think of error :D Yeah then it should be changed!
I always use print when developing for the speed but your idea is better!

Reply:
> I always use print when developing for the speed but your idea is better!

So do I! :D